chain/network/src/stats/rate_counter.rs

```diff
@@ -15,10 +15,13 @@
 use std::collections::VecDeque;
 use std::time::{Duration, SystemTime};
 
-const MINUTE_IN_MILLIS: u128 = 60_000;
+const MINUTE_IN_MILLIS: Duration = Duration::from_secs(60);
 
+/// Stores list of entries for `RateCounter`
 struct Entry {
+    /// bytes since last reset
     bytes: u64,
+    /// Time when we measured num `bytes` plus 1 minute.
```

Comment:
time when this entry should no longer be counted (which is currently set to time when we measured it plus 1 minute)

Reply:
I rewrote code to store current time. This will be much easier to understand.